Source/WebInspectorUI/UserInterface/Protocol/Legacy/iOS/26.0/InspectorBackendCommands.js

```diff
@@ -0,0 +1,570 @@
+InspectorBackend.registerCommand("Canvas.requestContent", null, [{"name": "canvasId", "type": "string"}], ["content"]);
+InspectorBackend.registerCommand("Canvas.requestClientNodes", ["page"], [{"name": "canvasId", "type": "string"}], ["clientNodeIds"]);
+InspectorBackend.registerCommand("Canvas.resolveContext", null, [{"name": "canvasId", "type": "string"}, {"name": "objectGroup", "type": "string", "optional": true}], ["object"]);
+InspectorBackend.registerCommand("Canvas.setRecordingAutoCaptureFrameCount", null, [{"name": "count", "type": "number"}], []);
+InspectorBackend.registerCommand("Canvas.startRecording", null, [{"name": "canvasId", "type": "string"}, {"name": "frameCount", "type": "number", "optional": true}, {"name": "memoryLimit", "type": "number", "optional": true}], []);
+InspectorBackend.registerCommand("Canvas.stopRecording", null, [{"name": "canvasId", "type": "string"}], []);
+InspectorBackend.registerCommand("Canvas.requestShaderSource", null, [{"name": "programId", "type": "string"}, {"name": "shaderType", "type": "string"}], ["source"]);
+InspectorBackend.registerCommand("Canvas.updateShader", null, [{"name": "programId", "type": "string"}, {"name": "shaderType", "type": "string"}, {"name": "source", "type": "string"}], []);
+InspectorBackend.registerCommand("Canvas.setShaderProgramDisabled", null, [{"name": "programId", "type": "string"}, {"name": "disabled", "type": "boolean"}], []);
+InspectorBackend.registerCommand("Canvas.setShaderProgramHighlighted", null, [{"name": "programId", "type": "string"}, {"name": "highlighted", "type": "boolean"}], []);
+InspectorBackend.registerEvent("Canvas.canvasAdded", null, ["canvas"]);
+InspectorBackend.registerEvent("Canvas.canvasRemoved", null, ["canvasId"]);
+InspectorBackend.registerEvent("Canvas.canvasSizeChanged", null, ["canvasId", "width", "height"]);
+InspectorBackend.registerEvent("Canvas.canvasMemoryChanged", null, ["canvasId", "memoryCost"]);
+InspectorBackend.registerEvent("Canvas.extensionEnabled", null, ["canvasId", "extension"]);
+InspectorBackend.registerEvent("Canvas.clientNodesChanged", ["page"], ["canvasId"]);
+InspectorBackend.registerEvent("Canvas.recordingStarted", null, ["canvasId", "initiator"]);
+InspectorBackend.registerEvent("Canvas.recordingProgress", null, ["canvasId", "frames", "bufferUsed"]);
+InspectorBackend.registerEvent("Canvas.recordingFinished", null, ["canvasId", "recording"]);
+InspectorBackend.registerEvent("Canvas.programCreated", null, ["shaderProgram"]);
+InspectorBackend.registerEvent("Canvas.programDeleted", null, ["programId"]);
+InspectorBackend.registerCanvasDispatcher = InspectorBackend.registerDispatcher.bind(InspectorBackend, "Canvas");
+InspectorBackend.activateDomain("Canvas", ["page", "web-page"]);
+
+// Console
+InspectorBackend.registerDomain("Console", ["itml", "javascript", "page", "service-worker", "worker"]);
+InspectorBackend.registerEnum("Console.ChannelLevel", {Off: "off", Basic: "basic", Verbose: "verbose"});
```

Comment:
there should still be a `ChannelSource` here, just without any mention of `AppCache`

Reply:
Yeah, the latest change was too broad: [diff](https://github.com/WebKit/WebKit/compare/b8610497e2eaa23c810b15688750dd308a321e3e..df5c1366be692dd75ddb39137870b1a3468aa1dc#diff-8e228f13a146e14a7ea22884f954ee8cf24644861c92ce53b126799a3b5589d3)